daml-lf/archive/da/daml_lf_1.proto

```diff
@@ -1016,18 +1042,24 @@ message DefTemplate {
 // Data type definition
 message DefDataType {
   message Fields {
-    repeated FieldWithType fields = 1; // *must be non-empty*
+    repeated FieldWithType fields = 1;
+  }
+
+  message EnumConstructor {
+    repeated string constructors = 1;
```

Comment:
If you're calling them constructors here, which I think is OK, then it should probably be the same in the enum pattern.